lib/squadster_web/controllers/auth_controller.ex

```diff
@@ -20,9 +20,8 @@ defmodule SquadsterWeb.SessionController do
   end
 
   def callback(%{assigns: %{ueberauth_auth: auth}} = conn, _params) do
-    case User.find_or_create(auth) do
-      {:ok, user} ->
-        redirect(conn, external: redirect_url(token: user.auth_token))
+    case Accounts.find_or_create_user(auth) do
+      {:ok, user}      -> redirect(conn, external: redirect_url(token: user.auth_token))
```

Comment:
this whitespace looks like a mess
maybe discard this changes
take a decision

Reply:
ok, why not